editor/connections_dialog.cpp

```diff
@@ -1203,58 +1203,68 @@ void ConnectionsDock::_slot_menu_about_to_popup() {
 }
 
 void ConnectionsDock::_tree_gui_input(const Ref<InputEvent> &p_event) {
-	// Handle Delete press.
-	if (ED_IS_SHORTCUT("connections_editor/disconnect", p_event)) {
-		TreeItem *item = tree->get_selected();
-		if (item && _get_item_type(*item) == TREE_ITEM_TYPE_CONNECTION) {
-			Connection connection = item->get_metadata(0);
-			_disconnect(connection);
-			update_tree();
+	const Ref<InputEventKey> &key = p_event;
+
+	if (key.is_valid() && key->is_pressed() && !key->is_echo()) {
+		// Handle Delete press.
+		if (ED_IS_SHORTCUT("connections_editor/disconnect", p_event)) {
+			TreeItem *item = tree->get_selected();
+			if (item && _get_item_type(*item) == TREE_ITEM_TYPE_CONNECTION) {
+				Connection connection = item->get_metadata(0);
+				_disconnect(connection);
+				update_tree();
+
+				// Stop the Delete input from propagating elsewhere.
+				accept_event();
+				return;
+			}
+			// Handle Ctrl+F press.
```

Comment:
This should be moved down, it is now part of the `connections_editor/disconnect` shortcut